xonsh/tools.py

```diff
@@ -35,15 +35,15 @@
 import threading
 import traceback
 import warnings
-import operator
+from distutils.version import LooseVersion
 
 # adding imports from further xonsh modules is discouraged to avoid circular
 # dependencies
 from xonsh import __version__
-from xonsh.lazyasd import LazyObject, LazyDict, lazyobject
-from xonsh.platform import (has_prompt_toolkit, scandir, DEFAULT_ENCODING,
-                            ON_LINUX, ON_WINDOWS, PYTHON_VERSION_INFO,
-                            expanduser, os_environ)
+from xonsh.lazyasd import LazyDict, LazyObject, lazyobject
+from xonsh.platform import (DEFAULT_ENCODING, ON_LINUX, ON_WINDOWS,
+                            PYTHON_VERSION_INFO, expanduser,
+                            has_prompt_toolkit, os_environ, scandir)
```

Comment:
and here and the lines above -- if we aren't adding anything then leave the imports alone